tests/handlers/aws/test_outputs.py

```diff
@@ -0,0 +1,135 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
minor. I would rename to `test_logstash.py` or something similar

I know that @girodav plans to split further `test_handler.py` so the more specific we are with the filename the more we can split giving the right hint just looking at the filename :)